frontend/src/components/Scoring.css

```diff
@@ -0,0 +1,63 @@
+body {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.FullScoring {

```

Comment:
css classes are kebab-case or eventually camelCase, but not PascalCase. Change this for all of your classes.